danilovid/second/task1.py.py

```diff
@@ -0,0 +1,12 @@
+"Використовуючи регулярні вирази, для поданого нижче тексту замініть кожне входження автомобільного номеру у форматі `АА 0000 BB` на текст `[номер АА]`. Виведіть результат."
+
+import re
+
```

Comment:
First of all, I would add validators and it is possible not to import the whole re. We only may use this for findall match and compile.